### DISCUSSION TITLE
Virtual Chassis Master to show all Interfaces

### DISCUSSION THREAD

**Discussion by digital-fortress** [nautobot/nautobot#6036] [Ideas]
Master device should show interfaces in Member devices as well. Can this be made possible

**Comment by jdrew82** [comment]
That is how it works. You should need a VirtualChassis created with a master Device created and assigned. Then assign the member Devices to the VirtualChassis and all of their Interfaces should show combined on the master Device.

**Reply by digital-fortress** [reply]
> That is how it works. You should need a VirtualChassis created with a master Device created and assigned. Then assign the member Devices to the VirtualChassis and all of their Interfaces should show combined on the master Device.

Thanks, did this feature break in new pre-release version ? I doubt, someone can test and confirm. I went back to my test system with (v2.2.5) and its working. (I am still doing POC so not a headache for me at the moment)

**Reply by glennmatthews** [reply]
Sounds like the issue just reported as #6045. We'll look into it.

**Comment by nkallergis** [comment]
What @jdrew82 said. You can see this in action [here](https://demo.nautobot.com/dcim/devices/60750778-9c40-4712-bbe0-655fd1742265/?tab=main).
